app/core/utils.js

```diff
@@ -301,7 +301,8 @@ utils.roundBgTarget = (value, units) => {
 
 utils.getTimezoneForDataProcessing = (data, queryParams) => {
   var timePrefsForTideline;
-  var mostRecentUpload = _.sortBy(_.filter(data, {type: 'upload'}), (d) => Date.parse(d.time)).reverse()[0];
+  // workaround a bug in upload timezone
+  var mostRecentUpload = _.sortBy(_.filter(data, {type: 'cbg'}), (d) => Date.parse(d.time)).reverse()[0];
```

Comment:
Does this change lead to a slower loading ?
There is far more cbg data than upload ones.

Reply:
good point 
we could even bypass it as we don't have any timezone in our objects. Let me switch to a food object